### PR TITLE
DIV-5762 - use java chart v2.6.0 and reduced duplicated env references

### PR DESCRIPTION
**Pull request by hosainnet**
# Description

Reduced duplicated config values by moving them to values.yaml
And using {{ .Values.global.environment }} placeholder
Also removed unused env variables (if any)

See 
* https://github.com/hmcts/chart-java/pull/61 
* https://hmcts-reform.slack.com/archives/CLE7A3SKV/p1568286603003800

Fixes #DIV-5762

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: charts/div-fps/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: Divorce - Fees and Payment Service
 name: div-fps
-version: 1.1.0
+version: 1.2.0
```
Comment (tchow8) [inline charts/div-fps/Chart.yaml]: You used 1.3.0 in div-dgs?

File: charts/div-fps/values.preview.template.yaml [hunk 2]
```diff
@@ -1,8 +1,4 @@
 java:
-    environment:
-        FEE_API_URL : "http://fees-register-api-aat.service.core-compute-aat.internal"
-        MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "30000"
```
Comment (qzhou-hmcts) [inline charts/div-fps/values.preview.template.yaml]: why MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE has been deleted?

Comment (hosainnet) [reply charts/div-fps/values.preview.template.yaml]: Sorry I meant to move it to `values.yaml` 👍

File: charts/div-fps/Chart.yaml (hunk 1 above)
Comment (hosainnet) [reply charts/div-fps/Chart.yaml]: Yep good spot, they should be on 1.2.0 except cos because that one matured faster :)

**Pull request merged**